filebeat/docs/inputs/input-filestream-reader-options.asciidoc

```diff
@@ -141,3 +141,92 @@ The default is 16384.
 
 The maximum number of bytes that a single log message can have. All bytes after
 `mesage_max_bytes` are discarded and not sent. The default is 10MB (10485760).
+
+[float]
+===== `parsers`
+
+This option expects a list of parsers the log line has to go through. 
+
+Avaliable parsers:
+- `multiline`
+- `ndjson`
+
+In this example a JSON object that spans 3 lines are aggregated into a single
+event and then parsed by `ndjson`.
+
+["source","yaml",subs="attributes"]
+----
+{beatname_lc}.inputs:
+- type: {type}
+  ...
+  parsers:
+    - multiline:
+      type: counter
+      lines_count: 3
+      keep_newline: false
+    - ndjson:
+      keys_under_root: true
+----
+
+See the available parser settings in detail below.
+
+[float]
+===== `multiline`
+
+Options that control how {beatname_uc} deals with log messages that span
+multiple lines. See <<multiline-examples>> for more information about
+configuring multiline options.
```

Comment:
@dedemorton do we have a link to the settings, or shall we copy the documentation from the logs input?

Reply:
The page "Multiline examples" is an independent page from the `log` input.